src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -1648,5 +1649,10 @@ public override void WriteToken(XmlWriter writer, SecurityToken token)
         {
             throw new NotImplementedException();
         }
+
+        /// <inheritdoc/>
+        /// No implementation since this class has been deprecated.
+        /// 
```

Comment:
redundant line

Reply:
We can't deprecate this class and will need an implementation here.

Reply:
The implementation is now added.